src/custom/pages/Claim/ClaimingStatus.tsx

```diff
@@ -146,6 +150,20 @@ export default function ClaimingStatus() {
         </AttemptFooter>
       )}
       {isSubmitted && chainId && lastClaimTx?.hash && <EnhancedTransactionLink tx={lastClaimTx} />}
+
+      {isFailure && (
+        <>
+          {chainId && lastClaimTx?.hash && <EnhancedTransactionLink tx={lastClaimTx} />}
+          <AttemptFooter>
+            <p>The claim transaction failed. Please check the network parameters and try again.</p>
+          </AttemptFooter>
+        </>
+      )}
+      {(isConfirmed || isFailure) && (
```

Comment:
I think the go back should only be there for the failure. 

For the success, we should have a "Change account" that does the reset. Otherwise, i can just go back and try to claim again, and I get:
![image](https://user-images.githubusercontent.com/2352112/151793616-cf9b6584-0c2d-42f1-8ed2-91182962d362.png)